pkg/avancement/service_manager.go

```diff
@@ -252,14 +252,8 @@ func createPullRequest(ctx context.Context, fromURL, toURL, newBranchName, commi
 
 	u, _ := url.Parse(toURL)
 	// take out ".git" at the end only if present
-	pathToUse := ""
-	// We take from index 1 as index 0 is a slash
-	lastFourChars := u.Path[len(u.Path)-4:]
-	if lastFourChars == ".git" {
-		pathToUse = u.Path[1 : len(u.Path)-4]
-	} else {
-		pathToUse = u.Path[1:]
-	}
+	// 1: is important so we don't have a leading slash
```

Comment:
Maybe 

```go
pathToUse := strings.TrimPrefix(strings.TrimSuffix(u.Path, ".git"), "/")
```
might be more obvious than the comment?

Reply:
+1, done and everything still working as expected